app/Http/Controllers/AuthController.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+
+class AuthController extends Controller
+{
+    public function store (Request $req)
+    {
+        $req->validate([
+            "first_name" => "required",
+            "last_name" => "required",
+            "email" => "required|email",
+            "password" => "required|min:6",
+            "re_enter_password" => "required|same:password"
+        ]);
+
+        // Saving the values in database.
+        $user=new User;
+        $user->first_name=$req->first_name;
+        $user->last_name=$req->last_name; 
+        $user->email=$req->email;
+        $user->password=$req->password;
+        $user->save();
+        return redirect('login');
+    }
+
+    public function check(Request $req)
+    {
+        // Validate Requests
+        $req->validate([
+            "email" => "required|email",
+            "password" => "required|min:6"
+        ]);
+
+        // Retrive the information from database.
+        $user_info = User::where('email','=',$req->email)->first();
```

Comment:
```suggestion
        $userInfo= User::where('email','=',$req->email)->first();
```
Camelcase convention for variables.